io.github.wereturtle.ghostwriter.json

```diff
@@ -0,0 +1,120 @@
+{
+    "app-id": "io.github.wereturtle.ghostwriter",
+    "base": "io.qt.qtwebkit.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "ghostwriter",
+    "rename-icon": "ghostwriter",
+    "rename-desktop-file": "ghostwriter.desktop",
+    "rename-appdata-file": "ghostwriter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
```

Comment:
What does it need network access for?

Reply:
Users sometimes use remote images, JavaScript files, etc. from the live HTML preview.